docs/docs/features/persist-state.mdx

```diff
@@ -80,3 +80,19 @@ export const persist = persistState(authStore, {
   storage: localForage,
 });
 ```
+
+## Excluding keys from saved state
+
+`excludeKeys()` operator can be used to exclude some keys from saved state:
```

Comment:
```suggestion
The `excludeKeys()` operator can be used to exclude keys from the state:
```

Reply:
Haven't fixed those yet. Was the merge intentional?

Reply:
No, please fix it

Reply:
Should I leave the operator name as is or do you want to rename it to `omitKeys`?

Reply:
leave it

Reply:
@NetanelBasal thank you very much for all the awesome open source libraries you have created and articles you have written 🙏

Reply:
Thanks